dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -554,7 +554,7 @@ class ScriptsControllerTest < ActionController::TestCase
       hideable_stages: 'on',
       project_widget_visible: 'on',
       student_detail_progress_view: 'on',
-      stage_extras_available: 'on',
+      lesson_extras_available: 'on',
```

Comment:
Should this be testing on or true?

Reply:
The values get sent in from the UI as "on" for the checkboxes:

![image](https://user-images.githubusercontent.com/6564873/81772828-b1ad7c80-949b-11ea-8a3c-6ffb5f52bcb9.png)